superset/daos/report.py

```diff
@@ -95,30 +95,6 @@ def find_by_database_ids(database_ids: list[int]) -> list[ReportSchedule]:
             .all()
         )
 
-    @classmethod
-    def delete(
-        cls,
-        items: ReportSchedule | list[ReportSchedule],
-        commit: bool = True,
-    ) -> None:
-        item_ids = [item.id for item in get_iterable(items)]
-        try:
-            # Clean owners secondary table
-            report_schedules = (
```

Comment:
I'm not really sure why the reports were refetched as opposed to using the `items` provides. Note after removing the need to remove the owners this logic simply reduced to the `BaseDAO.delete` method and thus is unnecessary.